src/gui/advancedsettings.cpp

```diff
@@ -335,6 +337,8 @@ void AdvancedSettings::loadAdvancedSettings()
     spinBoxCheckingMemUsage.setValue(session->checkingMemUsage());
     spinBoxCheckingMemUsage.setSuffix(tr(" MiB"));
     addRow(CHECKING_MEM_USAGE, tr("Outstanding memory when checking torrents"), &spinBoxCheckingMemUsage);
+    item(CHECKING_MEM_USAGE, PROPERTY)->setToolTip(QString("<body><p>%1</p></body>")
+            .arg(tr("Higher numbers give faster re-checks but uses more memory")));
```

Comment:
@thalieht:  Grammar cleanup: 

>Higher numbers give faster **rechecks** but **use** more memory. 

"Rechecks" is a valid word in English.  Also corrected the verb tense to be consistent with the nouns.

Reply:
Fixed.